inbox/events/microsoft/parse.py

```diff
@@ -562,6 +562,9 @@ def get_event_description(event: MsGraphEvent) -> Optional[str]:
     Returns:
         Plain text string with all the HTML removed
     """
+    if not event["body"]:
```

Comment:
This will also return `None` if `event["body"] is an empty string, no? I'm not sure if that could cause any issue, probably not. If it's intentional, maybe we could add a test case.

Reply:
It would be really weird if you could get `None | "" | dict` from Microsoft, but yeah it would be correct to get `None` in such case.